### ISSUE TITLE
UAA must wait for the database to be ready before starting

### ISSUE THREAD

**Issue by f0rmiga** [SUSE/kubecf#3]
Sometimes the UAA starts before the database is fully ready. When this happens, UAA never gets ready. A similar approach for waiting on the database exists on SCF v2.x.

**Comment by viovanov** [comment]
closing in favor of cloudfoundry-incubator/cf-operator#636
